fast64_internal/oot/oot_level_writer.py

```diff
@@ -86,6 +88,11 @@ def ootExportSceneToC(
         if exportSubdir == "":
             raise PluginError("Scene folder " + sceneName + " cannot be found in the ootSceneDirs list.")
 
+    roomObjects = [
+        obj for obj in bpy.data.objects if ((obj.parent == originalSceneObj) and (obj.ootEmptyType == "Room"))
+    ]
+    for roomObj, room in zip(roomObjects, scene.rooms.values()):
```

Comment:
I have no idea lol

Reply:
I figure it's pure luck, this will eventually lead to weird bugs

See `addRoom` for how roomObj are mapped to `OOTRoom`s

This is a weird operation because exporting goes top down from blender objs and props to C, and this goes back up (edits props too), I don't see how to fix this cleanly